data/map.txt

```diff
@@ -22490,7 +22490,8 @@ system Rutilicus
 	government Republic
 	attributes "dirt belt"
 	arrival 625
-	habitable 625
+	departure 2000
+	habitable 2000
```

Comment:
Is this just for testing? Provide a savefile with this change made to the system in the `changes`.